src/app/page.module.css

```diff
@@ -41,3 +41,17 @@
     transform: rotate(0deg);
   }
 }
+@media (min-width: 300px) and (max-width: 420px) {
+  .welcomecontainer {
+    margin: 1rem;
+    font-size: 2rem;
+    margin-top: 3rem;
+    margin-bottom: 5rem;
```

Comment:
redundant margin lines which leads to unreadable CSS, redefine to

`margin: 3rem, 1rem, 5rem, 1rem`
order of this shorthand property goes like
`margin: top, right, bottom, left`